api/v1/pinning.go

```diff
@@ -262,6 +262,20 @@ func (s *apiV1) handleAddPin(c echo.Context, u *util.User) error {
 		overwrite = true
 	}
 
+	// Get user storage capacity
+	usc, err := s.getUserStorageCapacity(u)
+	if err != nil {
+		return err
+	}
+
+	if !usc.ValidateThreshold() {
+		return &util.HttpError{
+			Code:    http.StatusBadRequest,
+			Reason:  util.ERR_CONTENT_SIZE_OVER_LIMIT,
+			Details: fmt.Sprintf("Reached Max Storage Threshold: %.2f TB. Please contact the Estuary Team for provisionning dedicated infrastruture if additonal storage is needed.", util.BytesToTB(usc.HardLimit)),
```

Comment:
Same as above

Reply:
added our slack channel info in error msg details